bpf/http_sock.c

```diff
@@ -53,7 +45,7 @@ int BPF_KRETPROBE(kretprobe_sock_alloc, struct socket *sock) {
         return 0;
     }
 
-    bpf_dbg_printk("=== sock alloc %llx ===", id);
+    //bpf_dbg_printk("=== sock alloc %llx ===", id);
```

Comment:
Did you forget to uncomment all these `bpf_dbg_printk`?

Reply:
Ah yeah! I was debugging an issue with system wide mode and there was too much output. I'll bring them back :)